pkgs/applications/misc/fitnesstrax/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub
+, glib
+, gtk3
+, lib
+, rustPlatform
+, wrapGAppsHook
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "fitnesstrax";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "luminescent-dreams";
+    repo = "fitnesstrax";
+    rev = "${name}-${version}";
```

Comment:
you should switch the `name` attr to `pname` at a lower level they are concatenated like `${pname}-${version}` to make `name` But it's not available at an attribute level. Currently the derivation name is just `fitnesstrax`.

Reply:
done!